elastic/beats
# ironbank: update the list of maintainers for the ironbank

## Proposed commit message

Update the list of maintainers for the ironbank, so it reflects the new org.

## Checklist




## Author's Checklist



## How to test this PR locally

```bash
cd x-pack/metricbeat
mage package
make -C ironbank package
mage ironbank
```

Then you will see the `.tar.gz` in the `build/distributions` folder for each subscribed beat, and `hardening_manifest.yaml` will contain the new up-to-date list of maintainers

## Related issues

https://github.com/elastic/beats/pull/32251 was the initial implementation

## Use cases



## Screenshots



## Logs
